OP_LOWERING_GUIDE.md

```diff
@@ -80,3 +80,5 @@ at::Tensor & wrapper_Scalar_lerp_(at::Tensor & self, const at::Tensor & end, con
 ```
 
 `lerp_.Scalar` will use our `lerp.Scalar` implementation without us providing explictly lowering.
+
+For each node we need to pass a `ir::OpKind` which is usually in the format of `at::aten::op_name` ([link to example](https://github.com/pytorch/xla/blob/5ce99bff336325feb41a982dc80299fb53166b29/torch_xla/csrc/ops/var_mean.cpp#L36)). You can find the `OpKind` definition in [aten_interned_strings.h](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/core/aten_interned_strings.h) or [interned_strings.h](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/core/interned_strings.h). If the aten symbol is missing, you can submit a PR like this [link](https://github.com/pytorch/pytorch/pull/36851) to PyTorch.
```

Comment:
nit, "an `ir::OpKind`". You can also do , 
```
Here is an [example] (https://github.com/pytorch/xla/blob/5ce99bff336325feb41a982dc80299fb53166b29/torch_xla/csrc/ops/var_mean.cpp#L36)).
```

Reply:
Done.